deploy-manage/deploy/cloud-on-k8s/install-using-yaml-manifest-quickstart.md

```diff
@@ -14,5 +17,79 @@ mapped_urls:
 
 % Use migrated content from existing pages that map to this page:
 
+% removed both
 % - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-install-yaml-manifests.md
-% - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-deploy-eck.md
\ No newline at end of file
+% - [ ] ./raw-migrated-files/cloud-on-k8s/cloud-on-k8s/k8s-deploy-eck.md
+
+This method is the quickest way to get started with ECK if you have full administrative access to the Kubernetes cluster. 
+
+During the installation, the following components are installed or updated:
+
+* `CustomResourceDefinition` objects for all supported resource types (Elasticsearch, Kibana, APM Server, Enterprise Search, Beats, Elastic Agent, Elastic Maps Server, and Logstash).
+* `Namespace` named `elastic-system` to hold all operator resources.
+* `ServiceAccount`, `ClusterRole` and `ClusterRoleBinding` to allow the operator to manage resources throughout the cluster.
+* `ValidatingWebhookConfiguration` to validate Elastic custom resources on admission.
+* `StatefulSet`, `ConfigMap`, `Secret` and `Service` in `elastic-system` namespace to run the operator application.
+
+## Prerequisites and considerations
+
+Before you begin, review the following prerequisites and recommendations:
+
+* For this quickstart guide, your Kubernetes cluster is assumed to be already up and running. Before you proceed with the ECK installation, make sure you check the [supported versions](/deploy-manage/deploy/cloud-on-k8s.md#k8s-supported).
```

Comment:
could simplify this. is there a word that applies to this group of items? "operator"? "orchestrator"? "platform"?

```suggestion
* You're running a Kubernetes cluster using a [supported platform](/deploy-manage/deploy/cloud-on-k8s.md#k8s-supported).
```